packages/nuxt/src/common/types.ts

```diff
@@ -111,4 +113,12 @@ export type SentryNuxtModuleOptions = {
    * @default false
    */
   experimental_basicServerTracing?: boolean;
+
+  /**
+   * Options to be passed directly to the Sentry Rollup Plugin (`@sentry/rollup-plugin`) and Sentry Vite Plugin (`@sentry/vite-plugin`) that ship with the Sentry Nuxt SDK.
+   * You can use this option to override any options the SDK passes to the Vite (for Nuxt) and Rollup (for Nitro) plugin.
+   *
+   * Please note that this option is unstable and may change in a breaking way in any release.
+   */
+  unstable_sentryBundlerPluginOptions?: SentryRollupPluginOptions & SentryVitePluginOptions;
```

Comment:
I used `Options` from `@sentry/bundler-plugin-core` first (as this is the type the other plugins inherit from) but this gave me the following build error:
```
 ERROR  The keyword 'interface' is reserved (4:0) in /[folders...]/sentry-javascript/node_modules/@sentry/bundler-plugin-core/dist/types/index.d.ts
 ```
The `Options` are defined as interface, but Rollup defines `interface` as [reserved name](https://github.com/rollup/rollup/blob/89a68c2a69eefaaf5544f83367f31d98360354ed/src/utils/RESERVED_NAMES.ts#L27). I think this is related but I couldn't figure out how to fix that.